include/envoy/http/header_map.h

```diff
@@ -384,6 +384,18 @@ class HeaderMap {
    */
   virtual void iterateReverse(ConstIterateCb cb, void* context) const PURE;
 
+  enum class Lookup { Found, NotFound, NotSupported };
+
+  /**
+   * Lookup one of the O(1) headers by key.
```

Comment:
It would be clearer to refer to these as something like "predefined inline header (see ALL_INLINE_HEADERS below)". Using the complexity alone elides the semantic info here.

Reply:
Changed naming.